pilot/pkg/features/pilot.go

```diff
@@ -115,6 +115,15 @@ var (
 		return time.Second * time.Duration(terminationDrainDurationVar.Get())
 	}
 
+	InboundCaptureAll = env.RegisterStringVar(
+		"ISTIO_META_INCLUDE_INBOUND_PORTS",
+		"*",
+		"The captured ports for inbound traffic. '*' for capture all.",
+	)
+	IsInboundCaptureAll = func() bool {
+		return InboundCaptureAll.Get() == "*"
+	}
+
```

Comment:
?? Why do we need this feature check for the newer proxies ?

Reply:
The idea is to allow use new proxy on old pilot during the upgrade to 1.3 . Otherwise the new proxy will hang the app...

Reply:
We do not officially support new proxy on old pilot. And the old pilot will not have this code. It will have a completely different code. So once again, how is this helpful?

Reply:
> We do not officially support new proxy on old pilot.

Good to know it! I can remove then

Reply:
wait, @howardjohn  I remember you run tests on pilot 1.2 and proxy 1.3. 
Should we endeavor to achieve the version compatibility(although we don't declare), or it's off the table?

Reply:
if we achieve it that's fine, I'm not sure we require it though. it's useful for downgrade

Reply:
Moved to pilot-agent.go
Delete it if it is proved safe in 1.3 release